sdk/storage/storage-queue/review/storage-queue.api.md

```diff
@@ -213,11 +224,16 @@ export type MessageIdDeleteResponse = MessageIdDeleteHeaders & {
 };
 
 // @public
-export interface MessageIdUpdateHeaders {
+export interface MessageIdUpdateExceptionHeaders {
     clientRequestId?: string;
-    date?: Date;
     // (undocumented)
     errorCode?: string;
+}
+
+// @public
+export interface MessageIdUpdateHeaders {
+    clientRequestId?: string;
```

Comment:
Same here, the errorCode seems to be gone

Reply:
Same as above